web/public/mgmt/config.yml

```diff
@@ -3074,6 +3074,15 @@ collections:
             fields:
               - { label: Beta Main Text, name: betaMainText, widget: string }
               - { label: Beta Modal Button Text, name: betaModalButtonText, widget: string }
+          - label: SEO and Accessibility Page Titles
```

Comment:
[pebble] How do you feel about moving the contents of `next-seo.config.js` to be defined here? We could still import `config` and set the values in that file.

This would give the content team the ability to modify the site name and description if necessary. Further gives them control to change the name of "The Navigator" when the time comes.

Reply:
I'mma look into this after lunch, I think that would be fine. Idk how easy it would be to setup given its a file in a different place that needs to be named as such to work with our SEO stuff but I'll explore a bit

Reply:
I did this just FYI